testsuite/features/build_validation/init_clients/sle15sp4_minion.feature

```diff
@@ -42,6 +42,13 @@ Feature: Bootstrap a SLES 15 SP4 Salt minion
     And I follow "Proxy" in the content area
     Then I should see "sle15sp4_minion" hostname
 
+  Scenario: Use correct kernel image on the SLES 15 SP4 minion
+    When I remove package "kernel-default-base" from this "sle15sp4_minion"
+    And I install package "kernel-default" on this "sle15sp4_minion"
```

Comment:
I wonder if it would be better to address this at cloud-init phase, when we deploy that node.

Reply:
Thank you, Oscar. I sadly did not think of `cloud-init`. I tried to do it with Salt in sumaform, but due to the necessary reboot afterwards, it was hard to not mess up everything, since it affect server, proxy and Minions.
I will take a look at `cloud-init` later today.